apps/web/src/app/vault/components/vault-items/vault-collection-row.component.ts

```
@@ -74,6 +57,22 @@ export class VaultCollectionRowComponent implements OnInit {
     return this.organizations.find((o) => o.id === this.collection.organizationId);
   }
 
+  get permissionText() {
+    if (this.collection.manage) {
+      if (this.flexibleCollectionsEnabled) {
+        return this.i18nService.t("canManage");
+      } else {
+        return this.i18nService.t("canEdit");
+      }
+    } else if (!this.collection.readOnly) {
+      return this.i18nService.t("canEdit");
+    } else if (this.collection.readOnly) {
+      return this.i18nService.t("canView");
```

This doesn't account for situations like "Can edit, except passwords". Presumably it should reflect all possible mappings per the access selector component (shown when managing collection permissions): https://github.com/bitwarden/clients/blob/master/apps/web/src/app/admin-console/organizations/shared/components/access-selector/access-selector.component.ts#L120-L123

Ideally it would share the same logic, rather than duplicating it. 

I suggest using [convertToSelectionView](https://github.com/bitwarden/clients/blob/master/apps/web/src/app/admin-console/organizations/shared/components/access-selector/access-selector.models.ts#L100-L107) to transform the collection object into the CollectionPermission enum, then the move the [permissionsList](https://github.com/bitwarden/clients/blob/master/apps/web/src/app/admin-console/organizations/shared/components/access-selector/access-selector.component.ts#L119-L124) into an exported function, and use that to get the i18n string.